atest/testdata/standard_libraries/operating_system/get_file.robot

```diff
@@ -169,8 +183,8 @@ Get And Check File
     Should Be Equal    ${content}    ${expected}
 
 Grep And Check File
-    [Arguments]    ${pattern}    ${expected}    ${test FILE}=${UTF-8 LONG FILE}
-    ${content} =    Grep File    ${test FILE}    ${pattern}
+    [Arguments]    ${pattern}    ${expected}    ${test FILE}=${UTF-8 LONG FILE}    ${regexp}=${False}
+    ${content} =    Grep File    ${test FILE}    ${pattern}    regexp=${regexp}
```

Comment:
Could also use `&{config}` both in the signature and when calling the keyword to support any kind of configuration. Now that there's only `regexp` option, this approach works as well.